orchagent/portsorch.cpp

```diff
@@ -5465,7 +5465,7 @@ bool PortsOrch::initializePort(Port &port)
         string hostTxReadyStr = hostTxReadyVal ? "true" : "false";
 
         SWSS_LOG_DEBUG("Received host_tx_ready current status: port_id: 0x%" PRIx64 " status: %s", port.m_port_id, hostTxReadyStr.c_str());
-        setHostTxReady(port.m_port_id, hostTxReadyStr);
+        m_portStateTable.hset(port.m_alias, "host_tx_ready", hostTxReadyStr);
```

Comment:
@noaOrMlnx please check above in this functin, port.m_port_id is being used ...so all those are also failing?

Reply:
@prgeor 
The issue here is not port.X, but the fact that inside setHostTxReady() function, we call getPort(). 
getPort cannot return the port object until port is initialize.

Reply:
@noaOrMlnx Can you please remove the getPort() from setHostTxReady()? This way all host_tx_ready change will be contained in one single function

Reply:
There is no need for portid inside setHostTxReady,

`PortsOrch::setHostTxReady(swss::Port, const std::string &status)` and modify the syslog to show only Alias (no need for port ID)